package.json

```diff
@@ -15,8 +15,8 @@
     "bunyan": "1.5.1",
     "cookie-parser": "1.3.5",
     "express": "4.19.2",
-    "express-body-schema": "github:Synpheros/express-body-schema#master",
-    "express-fileupload": "1.2.1",
+    "express-body-schema": "github:e-ucm/express-body-schema#master",
```

Comment:
Maybe wise to change this dependency to https://express-validator.github.io/docs/guides/schema-validation or any other wellknown / supported dependency that supports object validations

Reply:
adding issue for this one